max_depth: 4.0
fixed_frame: map
support_surfaces:
  dining_table: [
      [-0.001, -0.655, 0.690],
      [-0.698, -0.577, 0.690],
      [-0.769, -1.197, 0.690],
      [-0.071, -1.262, 0.690]
    ]
  kitchen_table: [
      [2.090, -0.819, 0.732],
      [0.927, -0.687, 0.732],
      [0.848, -1.370, 0.732],
      [2.004, -1.500, 0.732]
    ]
  kitchen_counter: [
      [3.773, -0.433, 0.900],
      [3.236, -0.364, 0.900],
      [3.121, -1.498, 0.900],
      [3.637, -1.560, 0.900]
    ]
